health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/pages/employee/my_bills.js

```
@@ -65,9 +80,102 @@ const MyBills = () => {
+    const getTotalAmount = (selectedBills) => {
+        if (!selectedBills || selectedBills.length === 0) return 0;
+        return selectedBills.reduce((total, bill) => total + (bill?.totalAmount || 0), 0);
+    }
 
-    useEffect(() => {
+    useEffect(async () => {
         if (BillData) {
+
+            const updateBillData = async () => {
+                BillData.bills.forEach(async (bill) => {
+                    const bill = BillData?.bills?.[0];
```

Biome (1.9.4)

[error] Shouldn't redeclare 'bill'. Consider to delete it or rename it.

'bill' is defined here:


(lint/suspicious/noRedeclare)